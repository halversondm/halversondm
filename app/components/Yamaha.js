/**
 * Created by Daniel on 6/26/2016.
 */
"use strict";

import React from "react";
import PhotoGallery from "./PhotoGallery";

const Yamaha = React.createClass({

  render() {
    return <div>
      <h2 className="text-primary">2004 Yamaha VStar 1100 Classic</h2>
      <p>In July 2004, I bought this Yamaha from Reno's Yamaha in Kansas City,
        Missouri. I upgraded the exhaust to Hard Krome, added a Corbin based
        Yamaha option seat, and added some chrome goodies from Yamaha.</p>
      <PhotoGallery perPage={4} totalPhotos={4} filePrefix="images/yamahaPhotos/photo" fileSuffix=".jpg" />
    </div>;
  }
});

export default Yamaha;
