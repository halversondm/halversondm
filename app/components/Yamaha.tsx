/**
 * Created by Daniel on 6/26/2016.
 */
import * as React from 'react'
import { PhotoGallery } from './PhotoGallery'
import { type ReactNode } from 'react'

export class Yamaha extends React.Component<unknown, unknown> {
  render (): ReactNode {
    return (
            <div>
                <h2 className="text-primary">2004 Yamaha VStar 1100 Classic</h2>
                <p>In July 2004, I bought this Yamaha from Reno&quot;s Yamaha in Kansas City,
                    Missouri. I upgraded the exhaust to Hard Krome, added a Corbin based
                    Yamaha option seat, and added some chrome goodies from Yamaha.</p>
                <PhotoGallery perPage={4} totalPhotos={4} filePrefix="images/yamahaPhotos/photo" fileSuffix=".jpg"/>
            </div>
    )
  }
}
