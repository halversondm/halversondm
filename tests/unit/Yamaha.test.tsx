import React from "react";
import { render } from '@testing-library/react';
import Yamaha from "../../app/components/Yamaha";

test("Yamaha component", () => {
    const unit = <Yamaha />;
    const component = render(unit);
    const tree = component.container;
    expect(tree.toString()).toMatchSnapshot();
});