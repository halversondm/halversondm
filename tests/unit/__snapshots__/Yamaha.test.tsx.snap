// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Yamaha component 1`] = `"<div><div><h2 class=\\"text-primary\\">2004 Yamaha VStar 1100 Classic</h2><p>In July 2004, I bought this Yamaha from Reno\\"s Yamaha in Kansas City, Missouri. I upgraded the exhaust to Hard Krome, added a Corbin based Yamaha option seat, and added some chrome goodies from Yamaha.</p><div id=\\"photoGallery\\"><ul class=\\"row photo-gallery\\"><li class=\\"col-md-2 col-lg-2 col-sm-3 col-xs-4\\"><img data-i=\\"0\\" class=\\"img-fluid\\" src=\\"images/yamahaPhotos/photo1.jpg\\"></li><li class=\\"col-md-2 col-lg-2 col-sm-3 col-xs-4\\"><img data-i=\\"1\\" class=\\"img-fluid\\" src=\\"images/yamahaPhotos/photo2.jpg\\"></li><li class=\\"col-md-2 col-lg-2 col-sm-3 col-xs-4\\"><img data-i=\\"2\\" class=\\"img-fluid\\" src=\\"images/yamahaPhotos/photo3.jpg\\"></li><li class=\\"col-md-2 col-lg-2 col-sm-3 col-xs-4\\"><img data-i=\\"3\\" class=\\"img-fluid\\" src=\\"images/yamahaPhotos/photo4.jpg\\"></li></ul><div class=\\"row\\"><div><button type=\\"button\\" data-id=\\"1\\" class=\\"btn btn-primary\\">Page 1</button></div></div></div></div></div>"`;
